src/GraphQL/Types/Scalars/ScalarGraphType.cs

```diff
@@ -4,7 +4,7 @@ namespace GraphQL.Types
 {
     public abstract class ScalarGraphType : GraphType
     {
-        public abstract object Serialize(object value);
+        public virtual object Serialize(object value) => ParseValue(value);
```

Comment:
Now only `DateGraphType`, `EnumerationGraphType`, `IdGraphType`, `TimeSpanMillisecondsGraphType` and `TimeSpanSecondsGraphType` override `Serialize`.